### PR TITLE
Add isort to CI

### PR DESCRIPTION
**Pull request by lymanepp**

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -2,11 +2,11 @@ aiohttp
 freezegun
 metar
 black==20.8b1
+click==8.0.4
```
Comment (lymanepp) [inline requirements-test.txt]: click-8.1.0 was published today and it contained a [breaking change](https://github.com/psf/black/issues/2964) for black

File: requirements-test.txt [hunk 2]
```diff
@@ -2,11 +2,11 @@ aiohttp
 freezegun
 metar
 black==20.8b1
+click<8.1
```
Comment (lymanepp) [inline requirements-test.txt]: click 8.1.0 was released today and is a breaking change for black 20.8b1. I will see if the latest version of black has the same problem.

Comment (lymanepp) [reply requirements-test.txt]: The latest version of black is also incompatible with click 8.1.0. But upgrading to latest is still a good idea IMO. This change needs to be merged to master as all future builds will be failing until a new version of black is released.

File: pynws/urls.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def gridpoints_forecast_hourly_url(wfo, x, y):
 
 
 def points_url(lat, lon):
-    """ Formats point metadata url."""
+    """Formats point metadata url."""
```
Comment (lymanepp) [inline pynws/urls.py:46]: The latest version of black caught this.

File: setup.cfg [hunk 4]
```diff
@@ -8,11 +8,9 @@ force_grid_wrap=0
 use_parentheses=True
 line_length=88
 indent = "    "
-# by default isort don't check module indexes
-not_skip = __init__.py
```
Comment (lymanepp) [inline setup.cfg:12]: Not needed with isort 5.x

File: setup.cfg [hunk 5]
```diff
@@ -8,11 +8,9 @@ force_grid_wrap=0
...
 # will group `import x` and `from x import` of the same module.
 force_sort_within_sections = true
-sections = FUTURE,STDLIB,INBETWEENS,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
```
Comment (lymanepp) [inline setup.cfg]: Not valid with isort 5.x

File: requirements-test.txt (hunk 2 above)
Comment (lymanepp) [reply requirements-test.txt]: Black 22.3.0 was just released. Upgraded and removed click version specification.

File: setup.cfg (hunk 4 above)
Comment (MatthewFlamm) [reply setup.cfg:12]: Can we instead use the new profile option in isort? 
 https://pycqa.github.io/isort/docs/configuration/black_compatibility.html#using-a-config-file-such-as-isortcfg

Comment (lymanepp) [reply setup.cfg:12]: > Can we instead use the new profile option in isort? https://pycqa.github.io/isort/docs/configuration/black_compatibility.html#using-a-config-file-such-as-isortcfg

Done

**Review by MatthewFlamm — APPROVED** [review]
I was planning on this and you beat me to it!

Would you prefer this to be merged first?  It seems to also address bugs in the CI with versions.

**Comment by lymanepp** [comment]
> I was planning on this and you beat me to it!
> 
> Would you prefer this to be merged first? It seems to also address bugs in the CI with versions.

Too many open PRs right now... I'd do https://github.com/MatthewFlamm/pynws/pull/83, then https://github.com/MatthewFlamm/pynws/pull/84 and finally https://github.com/MatthewFlamm/pynws/pull/82. I'll take care of the merge conflicts between.

**Comment by lymanepp** [comment]
https://github.com/MatthewFlamm/pynws/pull/84 is ready to be merged

**Pull request merged**